install/020-MqttBridgeConfigMap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
Just call it `020-ConfigMap.yaml` not need for the prefix. Otherwise you should have it for all the other YAMLs, which tbh is useless.